app/containers/Root.js

```diff
@@ -130,7 +110,7 @@ const mapStateToProps = state => ({
   hasWallets: walletSelectors.hasWallets(state),
   notifications: notificationSelectors.getNotificationState(state),
   theme: themeSelectors.currentThemeSettings(state),
-  isLoading: appSelectors.isLoading(state),
+  isLoading: appSelectors.isLoading(state) || lndSelectors.isStartingLnd(state),
```

Comment:
I think here you may need to make an exclusion for if `onboarding` is true - since we don't want the loading screen to flicker on/off when starting lnd in the onboarding process (this happens clicking "launch" to start a local wallet, and also when generating a new seed in the "Create new wallet" onboarding process.

Reply:
Actually, this probably won't work, because the state prop I was referring to (`state.onboarding.onboarding`) gets reset every time you restart the onboaridng process (ie, every time you enter the Onboarding Wizard).

Perhaps what we need is a new reducer for the home area so that we can track when it's in use, and then exclude triggering the loader based on this. Or, maybe exclude based on the current location? (though this feels a little off somehow).

Something like:

```js
  isLoading:
    (appSelectors.isLoading(state) || lndSelectors.isStartingLnd(state)) &&
    !onboardingSelectors.isOnboarding(state) &&
    !homeSelectors.isActive(state),
```

Reply:
I think I've found a way. Check the update

Reply:
Looks good @korhaliv